app/build.gradle

```diff
@@ -33,33 +33,38 @@ android {
     }
 
     buildTypes {
-        release {
+        debug {
+            shrinkResources false
+            minifyEnabled false
+            applicationIdSuffix ".debug"
+            manifestPlaceholders.isRaptorEnabled = "true"
+        }
+        release { // This isn't deployed to Google Play, but is used as a template for "real" releases like "beta"
```

Comment:
Is the goal to eventually use this for the release build? Having this build type just for the sake of having a template is going to be expensive. Or can the actual "only-release" build be ignored in `variantFilter`?

Reply:
That's a really good point, yeah. I'll represent the template separately, too, so it's not confusing what's reused and what's filtered :+1: